examples/material_gallery/lib/demo/menu_demo.dart

```diff
@@ -94,7 +94,7 @@ class MenuDemoState extends State<MenuDemo> {
                   child: new Text('Context menu item one')
                 ),
                 new PopupMenuItem(
-                  disabled: true,
+                  isDisabled: true,
```

Comment:
Can we change this to "enabled: false"? The Web has had a number of issues due to having booleans that are negative adjectives.